src/main/kotlin/org/axonframework/intellij/ide/plugin/creators/DefaultMessageCreator.kt

```diff
@@ -0,0 +1,32 @@
+package org.axonframework.intellij.ide.plugin.creators
+
+import com.intellij.psi.PsiElement
+import org.axonframework.intellij.ide.plugin.AxonIcons
+import org.axonframework.intellij.ide.plugin.api.Handler
+import org.axonframework.intellij.ide.plugin.api.MessageCreator
+import org.axonframework.intellij.ide.plugin.handlers.types.CommandHandler
+import org.axonframework.intellij.ide.plugin.handlers.types.EventSourcingHandler
+import org.axonframework.intellij.ide.plugin.handlers.types.SagaEventHandler
+import javax.swing.Icon
+
+data class DefaultMessageCreator(override val element: PsiElement, override val payloadFullyQualifiedName: String, override val parentHandler: Handler?) : MessageCreator {
+    override fun renderContainerText(): String? {
+        if (parentHandler is EventSourcingHandler) {
+            return "Side effect of EventSourcingHandler"
+        }
+        if (parentHandler is SagaEventHandler) {
+            return "Saga ${parentHandler.processingGroup}"
+        }
+        return null
```

Comment:
It is providing the container text in the list. This is the greyed out text next to the initial identifier. Such as here:
![image](https://user-images.githubusercontent.com/3278878/153386753-86358c17-c135-43dd-b7f9-8ae0d9bca918.png)

By default for creators, the white text shows the parent handler payload. So for example, if in a saga called `SomeSaga` the `SomeEvent` triggers the `SomeCommand`, the line marker in the aggregate will show the `SomeEvent` as origin. To qualify this more, we add 'Saga SomeSaga' to qualify this better. 

We could do this in a different or better way, I'm open for suggestions. 

For EventSourcingHandlers, it means it's a side-effect of a different event. It might not be necessary here, I added it more like a warning, we could remove it. 

In all other cases, I haven't found a use-case for the container-text here. If you know any, we could add it!